src/runtime-rs/crates/agent/src/kata/trans.rs

```diff
@@ -846,3 +847,35 @@ impl From<agent::OOMEvent> for OomEventResponse {
         }
     }
 }
+
+impl From<VolumeStatsRequest> for agent::VolumeStatsRequest {
+    fn from(from: VolumeStatsRequest) -> Self {
+        Self {
+            volume_guest_path: from.volume_guest_path,
+            unknown_fields: Default::default(),
+            cached_size: Default::default(),
```

Comment:
Nit: You could just say:

```suggestion
            ..Default::default()
```

Reply:
The structs defined in the previous lines didn't have this, should I apply this change to them all?